content/en/tracing/api_catalog/_index.md

```diff
@@ -0,0 +1,136 @@
+
+API endpoint
+: The URL for a server or a service, often through an HTTP, RESTful API interface, that implements the set of rules defined in the API. APIs operate through responses and requests -- a client makes a request and the API endpoint makes a response.<br /><br/>
+API endpoints in the API Catalog are comprised by the HTTP Method (for example, `GET`), the URL's Path (the structure of the location of the resource, such as `/payment/{shop_id}/purchase`) and the Service this resource serves (for example, `Payments`)<br /><br/>
+The API Catalog in beta supports only HTTP RESTful endpoints. 
+
+Public APIs
+: Publicly exposed API through the internet. An organization's way to expose services and apps to customers. In most cases, public APIs are used by web and mobile apps, but it's becoming more common to have public APIs that expose new revenue streams alongside web app. Customers expect APIs to be a central part of your product offering, and they set up workflows and automation that leverage those APIs.
+
+Private APIs
+: Also called internal APIs. Intended solely for internal use within a company or organization. They’re the most common type of API as they’re quick to develop and don’t need to be as robust as products intended for public consumption
+
+Partner APIs
+: Also called third party APIs. APIs that your app consumes that belong to a different organization (the third party). You leverage those APIs to provide your own service, and you are dependent on them for some features to work. For example, Paypal, Stripe, or other payment APIs, authentication providers including federated ones like Google or Facebook.
+
+## Exploring your endpoints
+
+The [API Catalog][1] view shows all endpoints in all environments in your Datadog organization. The performance data shown for each endpoint is scoped to the environments and time frame you select. You can browse and ask valuable questions by querying on different properties and metrics to get more precise results, using facets and tags for quick filtering.
+
+In the list table, **sort** by any of the columns by clicking a header. For example, click **P99** to see endpoints with the highest 99th percentile for latency.
+
+The table also shows **team ownership** for the API. This information is inherited from the associated service's definition in the [Service Catalog][2]. A service owner owns all of the endpoints that are connected to the service.
+
+To **filter the list** of endpoints or to search for a particular endpoint of interest, enter a query in the **Search** field. Search by service, path, or any other primary tag you wish. Or select a combination of facets on the left.
+
+To **scope the data** shown in the table, specify an environment, another primary tag (such as datacenter), and a time frame.
+
+{{< img src="tracing/api_catalog/api-catalog-scope.mp4" alt="Showing how changing scope settings changes the metrics shown in the API Catalog" video="true" >}}
```

Comment:
waiting for some clean up and adding the Sorting for latency and errors feature and then I will produce a better image here.
just before we release =)